insights/parsers/ls_edac_mc.py

```diff
@@ -0,0 +1,36 @@
+"""
+LsEdacMC - command ``ls -lan /sys/devices/system/edac/mc``
+============================================================
+
+The ``ls -lan /sys/devices/system/edac/mc`` command provides information for the listing of the
+``/sys/devices/system/edac/mc`` directory. See the ``FileListing`` class for a more complete description of the
+available features of the class.
+
+Sample ``ls -lan /sys/devices/system/edac/mc`` output::
+
+    /sys/devices/system/edac/mc:
+    total 90
+    drwxr-xr-x. 3 0 0 0 Jan 10 10:33 .
+    drwxr-xr-x. 3 0 0 0 Jan 10 10:33 ..
+    drwxr-xr-x. 2 0 0 0 Jan 10 10:33 power
+    drwxr-xr-x. 2 0 0 0 Jan 10 10:33 mc0
+    drwxr-xr-x. 2 0 0 0 Jan 10 10:33 mc1
+    drwxr-xr-x. 2 0 0 0 Jan 10 10:33 mc2
+
+Examples:
+    >>> '/sys/devices/system/edac/mc' in ls_edac_mc
+    True
+    >>> ls_edac_mc.dirs_of('/sys/devices/system/edac/mc') == ['.', '..', 'power', 'mc0', 'mc1', 'mc2']
+    True
```

Comment:
Please follow the [documentation guidelines](https://insights-core.readthedocs.io/en/latest/docs_guidelines.html) to include documentation for your parser in the build process.  Make sure that you test and review your documentation by running `make clean html_debug` in your docs directory before committing changes.  You are also missing the file `docs/shared_parsers_catalog/ls_edac_mc.rst` to include your parser docs in the build.

Reply:
@bfahr - ack, fixed the issue